app/src/ui/add-repository/add-existing-repository.tsx

```diff
@@ -172,7 +172,8 @@ export class AddExistingRepository extends React.Component<
 
   private showFilePicker = async () => {
     const window = remote.getCurrentWindow()
-    const directory = remote.dialog.showOpenDialog(window, {
+    // eslint-disable-next-line no-sync
+    const directory = remote.dialog.showOpenDialogSync(window, {
```

Comment:
i'm wondering if it makes more sense to `await` the non-sync version of these methods just to avoid the `eslint-disable-next-line` noise? the [`no-sync` rule is pretty literal](https://eslint.org/docs/rules/no-sync#rule-details) (it just flags methods that end in "sync"). i think functionally its the same though.